tests/tests_pytorch/loggers/test_logger.py

```diff
@@ -320,3 +321,57 @@ class _Test:
     )
     with pytest.raises(MisconfigurationException, match="Error while merging hparams"):
         trainer.fit(model, dm)
+
+
+@pytest.mark.parametrize("save_top_k", [0, 1, 2, 5])
+def test_scan_checkpoints(tmpdir, save_top_k: int):
```

Comment:
Can you re-write this test so that it doesn't need to train a model? You can instead mock the checkpoint_callback passed to `_scan_checkpoints`

Reply:
I re-wrote it, even though I am not sure it's exactly what you meant, as I am not familiar with mock. As it is now, no models are trained, just temporary files are created (I think that is needed by `_scan_checkpoints`).